tispark/tispark-user-guide.md

```diff
@@ -239,6 +194,58 @@ TiSparkR is a thin layer built to support the R language with TiSpark. Refer to
 
 TiSpark on PySpark is a Python package build to support the Python language with TiSpark. Refer to [this document](https://github.com/pingcap/tispark/blob/master/python/README.md) for usage.
 
+## Use TiSpark together with Hive
+TiSpark should be ok to use together with Hive. 
```

Comment:
```suggestion
You can use TiSpark together with Hive. 
```

Reply:
The "should be ok" is not clear. It seems that it might be ok and might be not ok.